themes/default/content/docs/get-started/aws/begin.md

```diff
@@ -99,7 +99,7 @@ $ export AWS_SECRET_ACCESS_KEY=<YOUR_SECRET_ACCESS_KEY>
 {{% /choosable %}}
 {{< /chooser >}}
 
-For additional information on setting and using AWS credentials, see [AWS Setup]({{< relref "/registry/packages/aws/installation-configuration" >}}).
+For information about AWS profiles, and configuration, see [AWS Setup]({{< relref "/registry/packages/aws/installation-configuration" >}}).
```

Comment:
Couldn’t we keep the same language and just also mention profiles?

```suggestion
For additional information on setting and using AWS credentials or profiles, see [AWS Setup]({{< relref "/registry/packages/aws/installation-configuration" >}}).
```

Reply:
@cnunciato what additional information is on that page that doesn't include profiles or credentials?

Reply:
@cnunciato do you specifically not want me to mention config? what credential information is there that's not in the guide and not in the profile content?